src/app.css

```diff
@@ -16,4 +16,19 @@ headerbar.selection-mode {
 
 button.tool, menubutton.tool {
```

Comment:
We don't need these anymore.

Reply:
(feel free to remove, if you don't I'll do that on some other branch eventually)